sdk/eventhub/sample.env

```diff
@@ -1,7 +1,12 @@
 # Basic info that will be required all the time.
```

Comment:
I don't think you meant to update this file.

Reply:
So, this file still shouldn't be modified.

However, you should add a `sample.env` to your package that lists the environment variables your tests need to run like we have here.

This will be `STORAGE_ACCOUNT_NAME` and `STORAGE_ACCOUNT_KEY`.